### PR TITLE
feat: query single bookmarks folder by id

### PR DESCRIPTION
**Pull request by sshanzel**

### CODE REVIEW

File: src/schema/bookmarks.ts [hunk 1]
```diff
@@ -517,6 +522,16 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
...
     },
+    bookmarkList: async (
+      _,
+      { id }: { id: string },
+      ctx: AuthContext,
+    ): Promise<GQLBookmarkList> => {
+      return ctx.con.getRepository(BookmarkList).findOneByOrFail({
+        id,
+        userId: ctx.userId,
+      });
```
Comment (capJavert) [inline src/schema/bookmarks.ts:533]: What do we use this query for, can we use read replica?

**Review by capJavert — APPROVED** [review]

**Review by AmarTrebinjac — APPROVED** [review]

File: src/schema/bookmarks.ts (hunk 1 above)
Comment (sshanzel) [reply src/schema/bookmarks.ts:533]: Just for the static props to get the relevant entity based on the query param. Definitely we can use read replica. Let me check some examples.

**Review by rebelchris — APPROVED** [review]

File: src/schema/bookmarks.ts [hunk 2]
```diff
@@ -517,6 +523,17 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
 
       return resolver(source, args, ctx, info);
     },
+    bookmarkList: async (
+      _,
+      { id }: { id: string },
+      ctx: AuthContext,
+    ): Promise<GQLBookmarkList> =>
+      queryReadReplica(ctx.con, async ({ queryRunner }) =>
```
Comment (capJavert) [inline src/schema/bookmarks.ts:531]: maybe use graphorm better, see https://github.com/dailydotdev/daily-api/pull/2516#discussion_r1877957934

File: src/schema/bookmarks.ts (hunk 1 above)
Comment (sshanzel) [reply src/schema/bookmarks.ts]: Do you have some samples for this, can't seem to make it work with graphorm.

Comment (sshanzel) [reply src/schema/bookmarks.ts]: Screenshot 2024-12-10 at 7 57 03 PM

Comment (sshanzel) [reply src/schema/bookmarks.ts]: Does this mean I will just pass the `ctx` and not use the `queryRunner` completely?

**Pull request merged**